Erro ao rodar make train-online

Subi o bot e fui capaz de treiná-lo e usá-lo.

Ao seguir o passo-a-passo sugerido no `readme.md`, contudo, este comando falhou ao rodar:

```
sudo docker-compose run --rm bot make train-online

Starting chatbot_actions_1 ... done
python3 -m rasa_core.train -d domain.yml -s data/stories -o models/dialogue --epochs 15 --online --nlu models/nlu/current/
usage: train.py [-h] {default,compare,interactive} ...
train.py: error: unrecognized arguments: --epochs 15 --online --nlu models/nlu/current/
Makefile:19: recipe for target 'train-online' failed
make: *** [train-online] Error 2
```

Vale notar que não é uma falha que impede o uso do bot/projeto.

--- a/bot/Makefile
+++ b/bot/Makefile
@@ -1,5 +1,6 @@
 train-nlu:
-	python3 -m rasa_nlu.train -c nlu_config.yml --fixed_model_name current --data data/intents/ -o models --project nlu --verbose
+	python3 -m rasa_nlu.train -c nlu_config.yml --fixed_model_name current \
+  --data data/intents/ -o models --project nlu --verbose
 
 train-core:
 	python3 train.py
@@ -10,13 +11,21 @@ run-rocketchat:
 	python3 run-rocketchat.py
 
 run-console:
-	python3 -m rasa_core.run -d models/dialogue -u models/nlu/current --debug --endpoints endpoints.yml
+	python3 -m rasa_core.run -d models/dialogue -u models/nlu/current \
+  --debug --endpoints endpoints.yml
 
 run-api:
-	python3 -m rasa_core.run -d models/dialogue -u models/nlu/current --endpoints endpoints.yml --debug --enable_api 
+	python3 -m rasa_core.run -d models/dialogue -u models/nlu/current \
+  --endpoints endpoints.yml --debug --enable_api 
 
 train-online:
-	python3 -m rasa_core.train -d domain.yml -s data/stories -o models/dialogue --epochs 15 --online --nlu models/nlu/current/
+	python -m rasa_core.train        \
+  interactive -o models/dialogue     \
+  -d domain.yml -c policy_config.yml \
+  -s data/stories                    \
+  --nlu models/nlu/current/          \
+  --endpoints endpoints.yml
+
 
 evaluate:
 	python3 -m rasa_core.evaluate -d models/dialogue -s data/core/

